### PR TITLE
Missing babel configuration in sw-precache example

### PR DESCRIPTION
**Pull request by jmaguirrei**
Missing configuration in package.json and .babelrc causes this example to be broken after installing and running the example.

### CODE REVIEW

File: examples/with-sw-precache/.babelrc [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+
+  "presets": [
+    [ "next/babel", { "preset-env": { "modules": "commonjs" }} ]
```
Comment (timneutkens) [inline examples/with-sw-precache/.babelrc]: This should *never* be done, only in the `test` environment.

File: examples/with-sw-precache/package.json [hunk 2]
```diff
@@ -10,5 +10,8 @@
     "react": "^16.0.0",
     "react-dom": "^16.0.0",
     "sw-precache-webpack-plugin": "^0.11.3"
+  },
+  "devDependencies": {
+    "@babel/preset-stage-0": "^7.0.0-beta.42"
```
Comment (timneutkens) [inline examples/with-sw-precache/package.json]: This doesn't add anything to the example

**Review by jmaguirrei — COMMENTED** [review]
.babelrc file now should be OK

File: examples/with-sw-precache/.babelrc [hunk 3]
```diff
@@ -0,0 +1,19 @@
+{
```
Comment (timneutkens) [inline examples/with-sw-precache/.babelrc]: There's no `"test"` script in this example so the babelrc shouldn't be here right

Comment (jmaguirrei) [reply examples/with-sw-precache/.babelrc]: Yes, you are right, I corrected it.

**Review by timneutkens — APPROVED** [review]

**Comment by timneutkens** [comment]
Thanks!

**Pull request merged**